lib/fastlane/plugin/android_sdk_update/actions/android_sdk_update_action.rb

```diff
@@ -52,6 +68,15 @@ def self.run(params)
         end
       end
 
+      def self.download_and_extract_sdk(download_url, sdk_path)
+        FastlaneCore::CommandExecutor.execute(
```

Comment:
Can you reformat the lines so it matches these on 61

Reply:
Certainly. Done.